Added RustConf to calendar

emmasofiah2o opened the pull request:
- created entry
- added description

To-do:
- find and add logo

marcoow reviewed (changes requested):
Can we use the SVG? That should have much smaller file size.

emmasofiah2o commented:
> Can we use the SVG? That should have much smaller file size.

I did use the SVG, changed colors, added a background, and exported it as a png. If you want me to export the image as an svg I can do that.
